chromium_src/content/browser/renderer_host/mixed_content_navigation_throttle.cc

```diff
@@ -0,0 +1,26 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "content/browser/renderer_host/mixed_content_navigation_throttle.h"
+
+#include "base/strings/string_util.h"
+
+#include "src/content/browser/renderer_host/mixed_content_navigation_throttle.cc"
+
+namespace content {
+
+// static
+bool MixedContentNavigationThrottle::DoesOriginSchemeRestrictMixedContent(
+    const url::Origin& origin) {
+  constexpr const char kOnion[] = ".onion";
```

Comment:
can this just reuse https://github.com/brave/brave-core/pull/15436/files#diff-ca5e905b331faa05f6ea79d8a922cb5c46833a68235e9e386a791f96b4d47477R21 ?

Reply:
No :(